tensorflow/python/training/monitored_session.py

```diff
@@ -575,7 +575,7 @@ def _close_internal(self, exception_type=None):
           ops.get_default_graph()._unsafe_unfinalize()  # pylint: disable=protected-access
 
   def _is_closed(self):
-    """Return True if the supervised session is closed.  For tests only.
+    """Return True if the monitored session is closed.  For tests only.
```

Comment:
I think the existing documentation is correct: it's the underlying supervised session in _coordinated_creator.tf_sess which is being checked, not self.